_releases/ocp-4.16-test-jobs-amd64.json

```diff
@@ -3,6 +3,9 @@
     {
       "prowJob": "periodic-ci-openshift-openshift-tests-private-release-4.16-automated-release-aws-ipi-private-shared-vpc-phz-sts-f360"
     },
+    {
+      "prowJob": "periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-private-fips-f4-disasterrecovery"
```

Comment:
@wangke19 we should add control flag, `optional: true`, otherwise this job will be considered as blocking job